packages/elements/src/components/ino-popover/readme.md

```diff
@@ -1,20 +1,117 @@
 # ino-popover
 
-A Popover is a dialog which is bound to a specific element and appears on top of the current page. It uses [tippy.js](https://atomiks.github.io/tippyjs/) to position the popover correctly.
+A Popover is a dialog which is bound to a specific element and appears next to it. Under the
+hood, [tippy.js](https://atomiks.github.io/tippyjs/) is used.
 
-The Popover and [Tooltip](https://elements.inovex.de/dist/latest/storybook/?path=/story/notification-ino-tooltip--default-usage) components are very similar.
-However, popovers are complex dialogs consisting of several HTML elements, while tooltips can only display plain text.
+The Popover
+and [Tooltip](https://elements.inovex.de/dist/latest/storybook/?path=/story/notification-ino-tooltip--default-usage)
+components are very similar. However, popovers are complex dialogs consisting of several HTML elements, while tooltips
+can only display plain text.
 
 ### Usage
 
 The component can be used as follows:
 
 ```html
-<ino-popover ino-for="<string>" ino-placement="<string" ino-trigger="<string>">
+
+<ino-popover ino-for="<string>" ino-placement="<string>" ino-trigger="<string>">
   Any desired HTML
 </ino-popover>
 ```
 
+#### Targets
+There are currently three ways to attach your popover to a component:
+
+1. Using the slot _(preferred)_:
+```html
+<ino-popover ino-trigger="click">
+    <ino-button slot="ino-popover-trigger">Click to show/hide</ino-button>
+    I'm the content of the popover
+</ino-popover>
```

Comment:
Maybe add for all the examples for the rendered tree where the popover is rendered in the end?

So 1 would be
ino-popover
  => button
  => Popover itself

etc.